tests/baselines/reference/tsc/declarationEmit/initial-build/when-same-version-is-referenced-through-source-and-another-symlinked-package.js

```diff
@@ -0,0 +1,37 @@
+//// [/lib/initial-buildOutput.txt]
+/lib/tsc -p src/plugin-one --listFiles
+/lib/lib.d.ts
+/src/plugin-one/node_modules/typescript-fsa/index.d.ts
+/src/plugin-one/action.ts
+/src/plugin-one/node_modules/plugin-two/node_modules/typescript-fsa/index.d.ts -> /src/plugin-one/node_modules/typescript-fsa/index.d.ts

```

Comment:
So, I didn't notice it before, but isn't `listFiles` consumed by some VS MSBuild API, and isn't this chance to include mappings with `->` probably going to break it?

Reply:
cc ... @uniqueiniquity  ?